src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -234,19 +248,21 @@ IntPtr LocalObjectToHandle(object input)
 
         private static PgoSchemaElem? ComputeLikelyClass(int index, Dictionary<IntPtr, object> handleToObject, PgoInstrumentationSchema[] nativeSchema, byte[] instrumentationData, CompilationModuleGroup compilationModuleGroup)
         {
-            // getLikelyClass will use two entries from the native schema table. There must be at least two present to avoid ovberruning the buffer
+            // getLikelyClasses will use two entries from the native schema table. There must be at least two present to avoid ovberruning the buffer
```

Comment:
```suggestion
            // getLikelyClasses will use two entries from the native schema table. There must be at least two present to avoid overruning the buffer
```